webclient/http1/src/main/java/io/helidon/webclient/http1/Http1ClientResponseImpl.java

```diff
@@ -166,10 +167,10 @@ public void close() {
                     } else if (entityLength == ENTITY_LENGTH_CHUNKED) {
                         if (hasTrailers) {
                             readTrailers();
```

Comment:
As far as I understand this, we should always check for `entityFullyRead` - and if not read, we should either consume the entity (e.g. throw it away), or close the connection (as we did originally).
Only when fully read should we release the connection (regardless of chunked encoding or content lenght). The only difference is that with chunked we may also need to read trailers...

Reply:
Fixed and trailered